test/jasmine/tests/click_test.js

```diff
@@ -817,6 +928,588 @@ describe('Test click interactions:', function() {
     });
 });
 
+
+describe('Test click interactions on a pie plot:', function() {
```

Comment:
While at it, @n-riesco would be mind resolving:

- https://github.com/plotly/plotly.js/issues/1456 and
- https://github.com/plotly/plotly.js/issues/902

This should be pretty easy. Thanks in advance! :white_check_mark: